lib/comgr/CMakeLists.txt

```diff
@@ -110,6 +118,14 @@ else()
   list(APPEND AMD_COMGR_PRIVATE_COMPILE_DEFINITIONS _HAS_EXCEPTIONS=0)
 endif()
 
+## Set library INTERFACE major, minor, patch version as compiler definitions
+list(APPEND AMD_COMGR_PRIVATE_COMPILE_DEFINITIONS
+     AMD_COMGR_INTERFACE_VERSION_MAJOR=${LIB_VERSION_MAJOR} )
+list(APPEND AMD_COMGR_PRIVATE_COMPILE_DEFINITIONS
+     AMD_COMGR_INTERFACE_VERSION_MINOR=${LIB_VERSION_MINOR} )
+list(APPEND AMD_COMGR_PRIVATE_COMPILE_DEFINITIONS
+     AMD_COMGR_INTERFACE_VERSION_PATCH=${LIB_VERSION_PATCH} )
+
```

Comment:
Can these be collapsed into a single `APPEND`, and the trailing whitespace before the `)` deleted?

Above you quote `"${ROCM_PATCH_VERSION}"`, and it is probably best to do that consistently here as well. Looking over the rest of the file I'm guilty of neglecting to quote scalar variables too, which I can fix in another patch.

Reply:
I remove the whitespace before ).
I didnt get next comment about the quote of ROCM_PATCH_VERSION, if it is about adding if (DEFINED ... )kind of guard, it is not required in my opinion. Please clarify on this.